### DISCUSSION TITLE
实际使用内存没有超过exec_mem_limit限制，但是报exec_mem_limit限制错误

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#14309] [A - General / Q&A]
exec_mem_limit是默认的2G，在查询的时候报了下面的错误，看错误信息说是使用了21100656，但是这个明显比限制的2147483648要小很多，请问这个是什么情况啊

INTERNAL: java.lang.RuntimeException: SQL execute error by datasource... java.sql.SQLException: errCode = 2, detailMessage = Memory exceed limit. New partitioned Aggregation in spill Backend: *.*.*.*, fragment: cfd01dbde2654f15-904a73d9c15533be Used: 21100656, Limit: 2147483648. You can change the limit by session variable exec_mem_limit.

**Comment by yuanyuan8983** [comment]
What version of Doris do you use?

**Reply by stgztsw** [reply]
1.0.0版本
